src/commands/renderSalesMessage.ts

```diff
@@ -1,20 +1,20 @@
 import { Message, MessageAttachment } from "discord.js"
-import { sales } from "types/sales"
+import { NftSales } from "types/sales"
```

Comment:
Already create function in `/utils/sales.ts` then fix the logic in `/utils/sales.ts`
If you want to use, import from "utils/sales" => can check if the function in `utils/sales.ts` is correct or not